Agents/BuildingMatchingAgent/BuildingMatchingAgent/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>uk.ac.cam.cares.jps</groupId>
+  <artifactId>BuildingMatchingAgent</artifactId>
+  <version>1.0.0</version>
+  <packaging>war</packaging>
+
+  <!-- Parent POM -->
+  <parent>
+    <groupId>uk.ac.cam.cares.jps</groupId>
+    <artifactId>jps-parent-pom</artifactId>
+    <version>1.0.0</version>
+  </parent>
+
+  <name>BuildingMatchingAgent Maven Webapp</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
This is woefully out of date. Please use something more recent.

Reply:
Addressed by commit [78556da](https://github.com/cambridge-cares/TheWorldAvatar/pull/616/commits/78556da93e569f13b79e10f6f7d0c027455378a4)